data/icon-overlay.js

```diff
@@ -240,12 +240,15 @@ function getTemplate() {
 
   containerEl.className = 'minvid__overlay__container';
   playIconEl.className = 'minvid__overlay__icon';
+  playIconEl.id = 'minvid__overlay__icon__play';
   playIconEl.title = 'Play Now';
-  addIconEl.className = 'minvid__overlay__icon__add';
+  addIconEl.className = 'minvid__overlay__icon';
+  addIconEl.id = 'minvid__overlay__icon__add';
```

Comment:
This was for the individual icon to have hover state, check the Gif below:
![apr-15-2017 00-30-12](https://cloud.githubusercontent.com/assets/8919308/25049152/dccc0bec-2172-11e7-94bd-21fc33618d3f.gif)